bootstrapvz/common/assets/init.d/expand-root

```diff
@@ -13,8 +13,19 @@
 prog=$(basename $0)
 logger="logger -t $prog"
 
-device_path="/dev/xvda"
+growpart=$(which growpart) || {
+	$logger "growpart was not found on PATH. Unable to expand size."
+	exit 0
+}
 
+root_index="0"
+
+$growpart /dev/xvda $root_index || {
```

Comment:
Wouldn't just calling `growpart` be fine here? We just established that it is on `$PATH`.